src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -99,7 +116,12 @@ private void convertParsedDataColumn(DataRepository dataRepository, KeyValuePair
          foreach (var value in columnAndData.Value)
          {
             if (!double.IsNaN(value.Lloq))
-               lloqValue = value;
+            {
+               if (lloqValue != null && !ValueComparer.AreValuesEqual(lloqValue.Lloq, value.Lloq))
+                  warningFlag = true;
```

Comment:
return true right away? why do you stil need to iterate?

Reply:
We need to iterate still because we want to give the user the opportunity to import even when there are inconsistencies so, we still need to populate the DataRepositories and show the user the option. We could have two functions, and then it would be as you propose but it would mean parse the data twice which is not very performant so if we agree this is the less frequent case, I believe it is best to keep iterating even when there is problems in case the user wants to import the data anyways instead of parsing the data twice always (one for checking and one for parsing)

Reply:
ok I see